less/components/nav.less

```diff
@@ -23,6 +23,20 @@
         opacity: 1;
       }
     }
+    .nav-lang-selector {
+      position: absolute;
```

Comment:
I initially used `margin-left: auto;`, but it doesn’t work in the mobile view. Do you prefer still using flexbox for desktop and position absolute for mobile?